docs/topics/testing/advanced.txt

```diff
@@ -873,8 +873,14 @@ project. You can see a report of this data by typing following command::
 Note that some Django code was executed while running tests, but it is not
 listed here because of the ``source`` flag passed to the previous command.
 
-For more options like annotated HTML listings detailing missed lines, see the
+If you are running tests in parallel using the ``--parallel`` option,
```

Comment:
I think this is worth adding. It could perhaps be just one sentence saying "use `--concurrency multiprocessing`". No need to explain `coverage combine`, but rather link to the `coverage run` docs, which details `--multiprocessing`, `coverage combine`, how to pass other args, etc. https://coverage.readthedocs.io/en/latest/cmd.html#execution-coverage-run